src/test/Shared/Services/Login.service.test.tsx

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright The Cryostat Authors
+ *
+ * The Universal Permissive License (UPL), Version 1.0
+ *
+ * Subject to the condition set forth below, permission is hereby granted to any
+ * person obtaining a copy of this software, associated documentation and/or data
+ * (collectively the "Software"), free of charge and under any and all copyright
+ * rights in the Software, and any and all patent rights owned or freely
+ * licensable by each licensor hereunder covering either (i) the unmodified
+ * Software as contributed to or provided by such licensor, or (ii) the Larger
+ * Works (as defined below), to deal in both
+ *
+ * (a) the Software, and
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import { AuthMethod, LoginService, SessionState } from '@app/Shared/Services/Login.service';
+import { TargetService } from '@app/Shared/Services/Target.service';
+import { AuthCredentials } from '@app/Shared/Services/AuthCredentials.service';
+import { ApiV2Response } from '@app/Shared/Services/Api.service';
+import { SettingsService } from '@app/Shared/Services/Settings.service';
+import { firstValueFrom, of, timeout } from 'rxjs';
+import { fromFetch } from 'rxjs/fetch';
+
+jest.mock('rxjs/fetch', () => {
+  return {
+    fromFetch: jest.fn((url: any, opts: any): any => {}),
```

Comment:
To resolve this eslint error, just change to 
```    
fromFetch: jest.fn((url: any, opts: any): any => undefined),
```

Running `yarn eslint:apply` should fix some other warnings to do with dependency import order, but thats just a bit of cleanup.

Reply:
Thanks, I wasn't aware of this!